src/Arith.jl

```diff
@@ -149,39 +146,71 @@ Base.iszero(x::CycloSum) = iszero(x.summands[1])  # isone(length(x.summands)) is
 struct CycloFrac{T} <: Cyclotomic{T}
 	numerator::CycloSum{T}
 	denominator::CycloSum{T}
-	function CycloFrac(numerator::CycloSum{T}, denominator::CycloSum{T}; simplify::Bool=true) where T<:NfPoly
+	exceptions::Set{ParameterException{T}}
+	function CycloFrac(numerator::CycloSum{T}, denominator::CycloSum{T}, exceptions::Set{ParameterException{T}}; simplify::Bool=true) where T<:NfPoly
 		if simplify
 			if iszero(numerator)
-				return new{T}(numerator, one(denominator))
+				return new{T}(numerator, one(denominator), exceptions)
 			elseif numerator == denominator
 				o=one(denominator)
-				return new{T}(o, o)
+				return new{T}(o, o, exceptions)
 			else
-				return new{T}(numerator, denominator)
+				return new{T}(numerator, denominator, exceptions)
 			end
 		else
-			return new{T}(numerator, denominator)
+			return new{T}(numerator, denominator, exceptions)
 		end
```

Comment:
Possible code simplifcation
```suggestion
		end
		return new{T}(numerator, denominator, exceptions)
```
... then remove the now empty `else`... next you can remove lines 157/158 (i.e. prior `else return ...`